src/main/webapp/app/overview/course-exams/course-exam-detail/course-exam-detail.component.html

```diff
@@ -0,0 +1,11 @@
+<div *ngIf="exam" (click)="openExam()">
+    <div class="row card-header text-white bg-secondary"><h4 class="card-title text-center">TITLE</h4></div>
```

Comment:
Why not insert the actual title here?

Reply:
The title is still not loaded or set yet, I'll communicate that with the server team and add it later

Reply:
The exam title should now be delivered to the client ;-)